app/services/maat_link_creator.rb

```diff
@@ -6,31 +6,41 @@ def initialize(laa_reference_id:)
   end
 
   def call
-    push_to_sqs unless laa_reference.dummy_maat_reference?
-    call_common_platform_endpoint
-    PastHearingsFetcherWorker.perform_at(30.seconds.from_now, Current.request_id, prosecution_case_id)
+    publish_laa_reference_to_queue
```

Comment:
This is less clear to me now. It looks like it will publish to queue even when `dummy_maat_reference`

Reply:
I put it back in 👌🏻

Reply:
I don't understand what has been put back in. I was thinking it wants to be 
`publish_laa_reference_to_queue unless laa_reference.dummy_maat_reference?`

Reply:
sorry, mixed up two things